libs/blocks/modal/modal.js

```diff
@@ -16,10 +17,25 @@ export function findDetails(hash, el) {
   return { id, path, isHash: hash === window.location.hash };
 }
 
+export function sendAnalytics(event) {
+  // eslint-disable-next-line no-underscore-dangle
+  window._satellite?.track('event', {
+    xdm: {},
+    data: {
+      web: { webInteraction: { name: event.type } },
```

Comment:
It might be a a good idea to check whether `event` exists before accessing its properties. A check at the start pf the method's logic or optional chaining here and on L26 should do it.